include/api.php

```diff
@@ -1355,66 +1311,20 @@ function api_status_show($type, $item_id = 0)
  */
 function api_users_show($type)
 {
-	$a = \get_app();
+	$a = \Friendica\BaseObject::getApp();
```

Comment:
Please don't add `\` at the beginning of namespaces. This could lead to problems during tests. I'm not sure if the absolute paths are really necessary.. I know there were a bug with `get_app()` and `\get_app()`, but I'm unsure if this is the right solution for it ..